app/src/components/tables/BlockTable/BlocksTable.test.tsx

```diff
@@ -0,0 +1,281 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { Table } from 'src/components';
+import {
+  getMockBlocks,
+  mockBlocksTableColumns,
+  getMockBlocksTableFooter,
+  getMockBlocksTableHeader,
+  mockBlocksTableOptions,
+} from 'src/mocks/mock-blocks-table';
+import { formatDate, truncateHash } from 'src/utils';
+import { render } from '../../../test-utils';
+
+const blocks = getMockBlocks();
+const mockBlocksTableFooter = getMockBlocksTableFooter();
+const mockBlocksTableHeader = getMockBlocksTableHeader();
+const totalPages = Math.ceil(
+  blocks.length / mockBlocksTableOptions.pagination.pageSize,
+);
+
+describe('BlocksTable', () => {
+  it('should render the BlocksTable', () => {
+    render(
+      <Table
+        header={mockBlocksTableHeader}
+        columns={mockBlocksTableColumns}
+        currentPageSize={mockBlocksTableOptions.pagination.pageSize}
+        isLastPage={totalPages === mockBlocksTableOptions.pagination.pageNum}
+        data={blocks}
+        footer={mockBlocksTableFooter}
+        tableBodyLoading={false}
+        placeholderData={{
```

Comment:
If these are all the same used across all the tests, can we assign this to a var and then reuse throughout?

Reply:
good call! fixed